torch/jit/frontend.py

```diff
@@ -193,7 +193,8 @@ def build_class_def(ctx, py_def, methods, self_name):
 
 def build_def(ctx, py_def, type_line, self_name=None):
     body = py_def.body
-    r = ctx.make_range(py_def.lineno, py_def.col_offset,
+    r = ctx.make_range(py_def.lineno + len(py_def.decorator_list),
```

Comment:
Without this change, the error highlight gets one line off, like:

```
RuntimeError:
Number of type annotations (2) did not match the number of function parameters (1):
at __scratch__/example.py:4:0
@torch.jit.script
~~~~~~~~~~~~~~~~~ <--- HERE
def f1(x):
    # type: (int, int) -> None
    pass

```

Essentially this changes `SourceRange` of `Decl` (and also `Def`) so that it excludes decorator lines.
As far as I can tell, `SourceRange` is only used for error reporting, so I thought this change is appropriate.
But, maybe I'm missing something (since I only grepped e.g. `def.range` or `decl.range`), so it would be great if someone could asses the effect of this change.